### PR TITLE
[DC-719] Add slider module

### PR DESCRIPTION
**Pull request by s-rubenstein**
### Jira Ticket: https://broadworkbench.atlassian.net/browse/DC-719

I saw the comments about doing a module after the merge - I figured I'd PR that separately and tag y'all on it!

### CODE REVIEW

File: src/components/common/Slider.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  width: 18,
+  zIndex: 0,
+};
+const trackHeight = 8;
+const trackStyle = { backgroundColor: colors.accent(0.5), height: trackHeight };
+const railStyle = { backgroundColor: colors.accent(0.4), height: trackHeight };
+
+const TerraSlider = (props) =>
+  h(Slider, {
+    handleStyle,
```
Comment (s-rubenstein) [inline src/components/common/Slider.ts:22]: I figure we probably want consistent styling across the app, so I pulled the styles in here.

File: src/pages/library/datasetBuilder/CohortEditor.ts [hunk 2]
```diff
@@ -1,10 +1,8 @@
-import 'rc-slider/assets/index.css';
-
 import _ from 'lodash/fp';
-import Slider from 'rc-slider';
 import React, { Fragment, useState } from 'react';
 import { div, h, h2, h3, strong } from 'react-hyperscript-helpers';
 import { ButtonOutline, ButtonPrimary, GroupedSelect, Link, Select } from 'src/components/common';
+import TerraSlider from 'src/components/common/Slider';
```
Comment (nawatts) [inline src/pages/library/datasetBuilder/CohortEditor.ts]: Nit: suggest naming this "Slider". That's in line with other components like "Select".

**Review by nawatts — APPROVED** [review]

File: src/pages/library/datasetBuilder/CohortEditor.ts (hunk 2 above)
Comment (cahrens) [reply src/pages/library/datasetBuilder/CohortEditor.ts]: I agree.

**Review by cahrens — APPROVED** [review]
Would prefer to see it named just "Slider". Otherwise LGTM.

**Pull request merged**